### DISCUSSION TITLE
Camera not started with motionplus.conf file only

### DISCUSSION THREAD

**Discussion by martin762** [Motion-Project/motionplus#41] [Q&A] [answered]
I have noticed that with the motionplus.conf file only (i.e no camerax.conf file) my webcam camera does not work (no light). I see the following in the logs:
webu_init_webcontrol: Unable to start MHD
With the same camera configs in a camera1.conf file camera comes on, streaming to localhost working, events recorded.

Is a requirement for a camerax.conf by design?

I am testing on a amd64 ubuntu20.04 OS but same issue seen on a raspberry pi with RaspberrypiOS32
Many thanks in advance.

**Comment by martin762** [comment]
A clarification to the above. concerning the Raspbery PI only. No change on AMD/Ubuntu. I do not see the "Unable to start MHD" statement on the PI and I can access the config tabs via localhost:8080.  No light on the webcam (USB webcam connected) and logs confirm camera not started. 
As soon as camera1 is enabled all is well.  In conclusion the  MHD reference is probably another issue and a red herring to my observation.

**Comment by Mr-Dave** [comment, marked as answer]
The message about MHD means that the port is in use.  Usually that means that another instance of Motion or Motionplus is already running and using the port.

The result of using only a motionplus.conf file is intentional and by design.  The motionplus.conf file provides the application defaults.  If no camera file is provided, then it assumes that you wish to use the web interface to add a camera after it has started.

**Reply by martin762** [reply]
Thank you for the advice concerning MHD. I did a pkill and that sorted it.  Also for the heads up about my observation concerning a motionplus.conf file without camera.conf files being by design. I will stop chasing my tail now. Cheers!
